src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialJsPage.java

```diff
@@ -52,4 +57,43 @@ public ContentReviewModule getReviewModule() {
         }
         return contentReviewModule;
     }
+
+    /**
+     * Checks if the element prepended to an article by the test-supplied JS is visible.
+     * @return boolean
+     */
+    public boolean isTestElementVisible() {
+        boolean isVisible = false;
+        try {
+            wait.forElementVisible(testElement, 3, 1);
+            isVisible = true;
+        } catch (TimeoutException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)